README.md

```diff
@@ -1,126 +1,188 @@
 # partner-charts
+This repository is reserved for partner charts in the Rancher's v2.5+ catalog.
 
-This is an experimental project to automate system charts and adding overlays files on top of it.
+As part of this catalog, all charts will benefit of a cloud native packaging system
+that directly references an upstream chart from a helm repo and automates applying 
+Rancher specific modifications and adding overlay files on top of it.
 
-Charts will have two categories: 
+## Requirements
+* Chart must be Helm v3 compatible
 
-1. Charts that Rancher created and maintained (**Rancher original**).
+    A common cause for Helm v3 incompatibility is using the `crd-install` hook provided in Helm 2 for 
+    the installation of CRDs. This hook was removed in v3 in favor of a `crds` directory where your 
+    crds should now reside. Templating and upgrading crds is also no longer supported.
```

Comment:
Upgrading is supported.

Reply:
Wait, are partner charts running through our chart split method or not ?

Reply:
Last I heard, partners will not run through our chart split method (Might need @deniseschannon to confirm). How they handle their CRDs is their responsibility.

Reply:
Ok then it would not be